inspirehep/modules/theme/jinja2filters.py

```diff
@@ -684,21 +684,19 @@ def format_date(datetext):
 @blueprint.app_template_filter()
 def find_collection_from_url(url):
     """Returns the collection based on the URL."""
-    if 'conferences' in url:
+    if 'cc=conferences' in url or '/conferences' in url:
```

Comment:
Would be great if this was actually done by properly parsing the URL query and path. In this way we would avoid unexpected behaviors.